doc/test_csd.py

```diff
@@ -0,0 +1,111 @@
+# PyCSD
+#
+# Copyright 2003-2010 Jürgen Kayser <rjk23@columbia.edu>
+# Copyright 2017 Federico Raimondo <federaimondo@gmail.com> and
+#                Denis A. Engemann <dengemann@gmail.com>
+#
+# The following code is a derivative work of the code from the CSD Toolbox,
+# which is licensed GPLv3. This code therefore is also licensed under the terms
+# of the GNU Public License, version 3.
```

Comment:
This is not the correct language :)

We got permission to relicense under BSD terms from the authors. Right @dengemann ?